admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss

```diff
@@ -46,7 +46,8 @@
     }
   }
 
-  .card,
+  .customer,
+  #orderProductsPanel,
```

Comment:
why Camel ?

Reply:
Well, this is a choice from the one who wrote the markup before me, if we change it, it's a BC break (maybe even our tests will fail)

But I can add a class on it so this won't suffer about an id change in the future (but maybe the class would change too anyway)

I think it's not that important :)